public/layouts/infopages/info.scss

```diff
@@ -0,0 +1,22 @@
+.admin {
+	padding:30px;
+}
+
+ul.status {
+  list-style-type: none;
+  padding: 0;	
+}
+
+.status label {
```

Comment:
the `label` here should have its own class and not rely on it being a child of `.status`